packaging/build_cmake.sh

```diff
@@ -45,6 +45,8 @@ else
     yum install -y libpng-devel libjpeg-turbo-devel
 fi
 
+source .circleci/unittest/windows/scripts/set_cuda_envs.sh
+
```

Comment:
This would be executed from both Windows and Linux build environment, which seems wrong